brickstrap.sh

```diff
@@ -428,15 +428,71 @@ function brp_preseed_debconf() {
     rm "$(br_rootfs_dir)/tmp/debconfseed.txt"
 }
 
+#
+# Ensure that a 'usable' /etc/shells file exists for the add-shell utility.
```

Comment:
I have never had a problem with this in wheezy/jessie. Is this just a problem in testing/unstable?